packages/whimsy-writer/vite.config.ts

```
@@ -0,0 +1,67 @@
+  for (let i = 0; i < linesIn.length; i++) {
+    const line = linesIn[i];
+    if (
+      [
+        `Cannot find name 'Passage'`,
+        `Cannot find name 'PassageName'`,
+        `Cannot find name 'PassageMap'`,
+        `Cannot find name 'Variables'`,
+      ].some((ignore) => line.includes(ignore))
+    ) {
+      i += 3;
+      continue;
```

Biome

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)